
velocityX: 0.2
velocityY: 0.0
velocityYaw: 0.0 
footHeight: 0.08 
bodyHeight: 0.26 